R/utils.R

```diff
@@ -110,4 +110,20 @@ compute_mz_sd <- function(feature_groups) {
     }
   }
   return(mz_sd)
+}
+
+#' @export
+get_num_workers <- function() {
```

Comment:
I suppose we can use this function also in other tests.

Reply:
yes - we can make an issue for it.